calculators/candidate_opposing_expenditures.rb

```diff
@@ -0,0 +1,52 @@
+class CandidateOpposingExpenditure
+  def initialize(candidates: [], ballot_measures: [], committees: [])
+    @committees = committees
+    @candidates = candidates
+  end
+
+  def fetch
+    # Get the total expenditures against candidates by date.
+    expenditures = ActiveRecord::Base.connection.execute(<<-SQL)
+      SELECT "FPPC" AS "Filer_ID", "Filer_NamL", "Exp_Date",
+      sum("Amount") AS total
+      FROM (
+        SELECT "Filer_NamL", "Exp_Date",
+          "Cand_NamF", "Cand_NamL", "Amount"
+        FROM "efile_COAK_2016_496"
+        WHERE "Cand_NamL" IS NOT NULL
+        AND "Sup_Opp_Cd" = 'O'
+        UNION ALL
+        SELECT "Filer_NamL", "Expn_Date" as "Exp_Date", "Cand_NamF", "Cand_NamL", "Amount"
+        FROM "efile_COAK_2016_E-Expenditure"
+        WHERE "Cand_NamL" IS NOT NULL
+        AND "Sup_Opp_Cd" = 'O'
+      ) AS U,
+      "oakland_candidates"
+      WHERE LOWER(TRIM(CONCAT("Cand_NamF", ' ', "Cand_NamL"))) = LOWER("Candidate")
```

Comment:
Is this going to break in some cases? It seems pretty flaky in case there are other parts of a candidate's name like their suffix or title or something.